micrometer-core/src/main/java/io/micrometer/core/aop/TimedAspect.java

```diff
@@ -274,6 +274,7 @@ private void record(ProceedingJoinPoint pjp, Timed timed, String metricName, Tim
 
     private Timer.Builder recordBuilder(ProceedingJoinPoint pjp, Timed timed, String metricName,
             String exceptionClass) {
+        @SuppressWarnings("NullTernary")
```

Comment:
This one was a bit odd. I'm not sure it's actually a problem. We're passing `null` to a primitive varargs, which I don't think has the problem as a non-varargs primitive like shown in [the bug pattern](https://errorprone.info/bugpattern/NullTernary).